README.md

```diff
@@ -5,6 +5,7 @@ A color temperature setting library and CLI that operates in a similar way to f.
 The command line app automatically determines your location using GeoIP and adjusts the color temperature depending on time.
 
 ```sh
+$ go get -d github.com/d4l3k/go-sct/sct
```

Comment:
The commit is here https://github.com/faf0/go-sct/commit/f30b03ba0527a32a099a51297d0ef49dc0478b35